oap-server/server-bootstrap/src/main/resources/lal/envoy-als.yaml

```
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rules:
+  - name: envoy-als
+    dsl: |
+      filter {
+        json {
+        }
+        // only collect abnormal logs (http status code >= 300, or commonProperties?.responseFlags is not empty)
+        if (parsed?.response?.responseCode < 300 && !parsed?.commonProperties?.responseFlags) {
```

```suggestion
        if (parsed?.response?.responseCode < 400 && !parsed?.commonProperties?.responseFlags) {
```
What do you think about 300 VS 400? @hanahmily @kezhenxu94 Do things like 302 matter?

---

I'd say >=300 <400 is normal cause the proxy will follow the redirect URL (query against the URL returned by 301, 302, or similar)